07-generic_bookworms/final/books/book_test.go

```diff
@@ -0,0 +1,80 @@
+package books_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
I would avoid such external packages. The general recommendation is to avoid assert/require like libraries.